src/components/Network.tsx

```diff
@@ -79,7 +79,7 @@ export const NetworkSides = () => {
   const { sourceChainDetails, targetChainDetails } = useSourceTarget();
   return (
     <Box marginY={2} className={classes.main}>
-      <Box>{sourceChainDetails.sourceChain}</Box>
+      <Box p>{sourceChainDetails.sourceChain}</Box>
```

Comment:
This is not correct. Probably you meant: <Box component="p"> ?

Reply:
So I honestly how this is supposed to work, but it's not about the component type, but rather about using `padding` from `theme`.
There are some details here: https://material-ui.com/system/spacing/#notation

I initially removed that cause the prop-validation complaints it's not a valid prop, but well, it looks like shit when removed, hence I guess we need it.
The error can be dealt with via #114